### ISSUE TITLE
MvxRadioGroup binding fails in Release after updating to 4.4.0

### ISSUE THREAD

**Issue by tbalcom** [MvvmCross/MvvmCross#1650]
I updated MvvmCross from 4.23 to 4.40 then I noticed the binding for MvxRadioGroup was no longer working in Release mode. It works as expected in Debug mode.  It used to work as expected in Release mode before I updated to version 4.40.

### Steps to reproduce

1. Add MvxRadioGroup in AXML:
```xml
<MvxRadioGroup
  android:id="@+id/hazmat_radiogroup"
  android:layout_width="wrap_content"
  android:layout_height="wrap_content"
  local:MvxItemTemplate="@layout/item_radio"
  local:MvxBind="ItemsSource HazardousMaterialsRadioItems; SelectedItem HazardousMaterials;"
  android:orientation="horizontal" />
```

2. Add `HazardousMaterialsRadioItems` and `HazardousMaterials` to ViewModel:

```C#
    public class RadioItem
    {
        public RadioItem(string caption)
        {
            Caption = caption;
        }
        public string Caption { get; }

        public override string ToString()
        {
            return Caption;
        }

        public override bool Equals(object obj)
        {
            var rhs = obj as RadioItem;
            if (rhs == null)
                return false;
            return rhs.Caption == Caption;
        }

        public override int GetHashCode()
        {
            return Caption?.GetHashCode() ?? 0;
        }

        public static RadioItem Yes = new RadioItem("Yes");
        public static RadioItem No = new RadioItem("No");
        public static RadioItem Loaded = new RadioItem("Loaded");
        public static RadioItem Empty = new RadioItem("Empty");
    }
```

```C#
        private ObservableCollection<RadioItem> _radioItems = new ObservableCollection<RadioItem>
        {
            RadioItem.Yes,
            RadioItem.No
        };
        public ObservableCollection<RadioItem> HazardousMaterialsRadioItems
        {
            get { return _radioItems; }
            set { SetProperty(ref _radioItems, value); }
        }

        private RadioItem _hazardousMaterials = RadioItem.Empty;
        public RadioItem HazardousMaterials
        {
            get { return _hazardousMaterials; }
            set
            {
                SetProperty(ref _hazardousMaterials, value);
                DoneCommand.RaiseCanExecuteChanged();
            }
        }
```

3. Run the program in Release mode and tap one of the RadioButtons. 
The following exception is thrown when you tap on the RadioButton:

```
Exception thrown during the view binding ArgumentNullException: missing source event info in MvxWeakEventSubscription Parameter name: sourceEventInfo
at MvvmCross.Platform.WeakSubscription.MvxWeakEventSubscription`2[TSource,TEventArgs]..ctor (MvvmCross.Droid.Support.V7.AppCompat.Widget.MvxAppCompatRadioGroup source, System.Reflection.EventInfo sourceEventInfo, System.EventHandler`1[TEventArgs] targetEventHandler) [0x00017] in <4ddde23419c5494288c799fcdbb0f189>:0 
at MvvmCross.Platform.WeakSubscription.MvxWeakEventSubscription`2[TSource,TEventArgs]..ctor (MvvmCross.Droid.Support.V7.AppCompat.Widget.MvxAppCompatRadioGroup source, System.String sourceEventName, System.EventHandler`1[TEventArgs] targetEventHandler) [0x00012] in <4ddde23419c5494288c799fcdbb0f189>:0 
at MvvmCross.Platform.WeakSubscription.MvxWeakSubscriptionExtensionMethods.WeakSubscribe[TSource,TEventArgs] (TSource source, System.String eventName, System.EventHandler`1[TEventArgs] eventHandler) [0x00000] in <4ddde23419c5494288c799fcdbb0f189>:0 
at MvvmCross.Droid.Support.V7.AppCompat.Target.MvxAppCompatRadioGroupSelectedItemBinding..ctor (MvvmCross.Droid.Support.V7.AppCompat.Widget.MvxAppCompatRadioGroup radioGroup) [0x00007] in <76c7f3cdfddd459898d814608e83832e>:0 
at MvvmCross.Droid.Support.V7.AppCompat.MvxAppCompatSetupHelper+<>c.<FillTargetFactories>b__0_1 (MvvmCross.Droid.Support.V7.AppCompat.Widget.MvxAppCompatRadioGroup radioGroup) [0x00000] in <76c7f3cdfddd459898d814608e83832e>:0 
at MvvmCross.Binding.Bindings.Target.Construction.MvxCustomBindingFactory`1[TTarget].CreateBinding (System.Object target, System.String targetName) [0x00027] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at MvvmCross.Binding.Bindings.Target.Construction.MvxTargetBindingFactoryRegistry.TryCreateSpecificFactoryBinding (System.Object target, System.String targetName, MvvmCross.Binding.Bindings.Target.IMvxTargetBinding& binding) [0x00019] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at MvvmCross.Binding.Bindings.Target.Construction.MvxTargetBindingFactoryRegistry.CreateBinding (System.Object target, System.String targetName) [0x00000] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at MvvmCross.Binding.Bindings.MvxFullBinding.CreateTargetBinding (System.Object target) [0x00013] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at MvvmCross.Binding.Bindings.MvxFullBinding..ctor (MvvmCross.Binding.MvxBindingRequest bindingRequest) [0x0002f] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at MvvmCross.Binding.Binders.MvxFromTextBinder.BindSingle (MvvmCross.Binding.MvxBindingRequest bindingRequest) [0x00000] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at MvvmCross.Binding.Binders.MvxFromTextBinder+<>c__DisplayClass2_0.<Bind>b__0 (MvvmCross.Binding.Bindings.MvxBindingDescription description) [0x00018] in <f9aae2b329aa4d8aa76dab4fe82e828d>:0 
at System.Linq.Enumerable+<CombineSelectors>c__AnonStorey1D`3[TSource,TMiddle,TResult].<>m__0 (TSource x) [0x00012] in <59b4bbd3eba24f208466668aef8a9bbb>:0 
at System.Linq.Enumerable+<CombineSelectors>c__AnonStorey1D`3[TSource,TMiddle,TResult].<>m__0 (TSource x) [0x00000] in <59b4bbd3eba24f208466668aef8a9bbb>:0 
at (wrapper delegate-invoke) System.Func`2[System.Collections.Generic.KeyValuePair`2[System.String,MvvmCross.Binding.Parse.Binding.MvxSerializableBindingDescription],System.Collections.Generic.KeyValuePair`2[System.Object,MvvmCross.Binding.Bindings.IMvxUpdateableBinding]]:invoke_TResult_T (System.Collections.Generic.KeyValuePair`2<string, MvvmCross.Binding.Parse.Binding.MvxSerializableBindingDescription>)
at System.Linq.Enumerable+WhereSelectEnumerableIterator`2[TSource,TResult].MoveNext () [0x00064] in <59b4bbd3eba24f208466668aef8a9bbb>:0 
at System.Collections.Generic.List`1[T].InsertRange (System.Int32 index, System.Collections.Generic.IEnumerable`1[T] collection) [0x000ff] in <bd1d7074fda4441aab7f9239e9e5ef1b>:0 
at System.Collections.Generic.List`1[T].AddRange (System.Collections.Generic.IEnumerable`1[T] collection) [0x00000] in <bd1d7074fda4441aab7f9239e9e5ef1b>:0 
at MvvmCross.Bindin
```

I've tried adding `RadioButton` to the LinkerPleaseInclude.cs file but it's made no difference:
```C#
        public void Include(RadioButton radioButton)
        {
            radioButton.CheckedChange += (sender, args) => radioButton.Checked = !radioButton.Checked;
        }
```
### Expected behavior
The `HazardousMaterials`  ViewModel property should be updated automatically when the user taps on the radio button. It was doing this in MvvmCross versions 4.23 and earlier.

### Actual behavior
ArgumentNullException: missing source event info in MvxWeakEventSubscription Parameter name: sourceEventInfo is thrown when I tap on the RadioButton.  The `HazardousMaterials` ViewModel property is not updated.

### Configuration

**Version:** 4.40
**Platform:** Android

**Comment by alexanderhanssen** [comment]
I noticed the same thing, works in debug but not in release mode. Got the same exception. Had to do a workaround by using RadioGroup instead of MvxRadioGroup and added RadioButtons with Checked binding on some bool properties.

**Comment by tbalcom** [comment]
Looks like this was just fixed 3 days ago in 3f6f7b74e996c5341c315e8e81e440310bba240d.  You need to add an "Include" method for both `RadioGroup` and `RadioButton`.
